internal/dockerhub/dockerhub.go

```diff
@@ -10,7 +10,34 @@ import (
 
 func GetLatestBitswanGitopsVersion() (string, error) {
 	// Get the latest version of the bitswan-gitops image by looking it up on dockerhub
-	getLatestVersionUrl := "https://hub.docker.com/v2/repositories/bitswan/pipeline-runtime-environment/tags/"
+	getLatestVersionUrl := "https://hub.docker.com/v2/repositories/bitswan/gitops/tags/"
+	resp, err := http.Get(getLatestVersionUrl)
```

Comment:
This smells of code duplication! Look below. Its almost the same.

Reply:
Fixed